### PR TITLE
Added date placeholder.

### PR DESCRIPTION
**Pull request by ycpei**
See #359.

Let me know if there are any problems.

### CODE REVIEW

File: autoload/vimwiki/html.vim [hunk 1]
```diff
@@ -1483,11 +1505,13 @@ function! s:convert_file(path_html, wikifile) "{{{
     call extend(ldest, lines)
 
     let title = s:process_title(placeholders, fnamemodify(a:wikifile, ":t:r"))
+    let date = s:process_date(placeholders, "")
```
Comment (EinfachToll) [inline autoload/vimwiki/html.vim]: Today's date as a default value would make a lot of sense.

Comment (ycpei) [inline autoload/vimwiki/html.vim]: Good point. I have made the change :)

**Comment by EinfachToll** [comment]
Many thanks!

**Comment by ycpei** [comment]
Thanks for merging!

**Pull request merged**